server/chat/backend/agent/tools/bitbucket/utils.py

```
@@ -79,36 +68,28 @@ def is_bitbucket_connected(user_id: str) -> bool:
+        selection = get_credentials_from_db(user_id, "bitbucket_workspace_selection") or {}
+
+        repositories = selection.get("repositories") or []
+        repo_data = next(
+            (r for r in repositories if isinstance(r, dict) and r.get("slug") == repo_slug),
+            None,
+        )
+        if repo_data:
+            mainbranch = repo_data.get("mainbranch")
+            if mainbranch:
+                branch = mainbranch.get("name")
+    except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)